cap/modules/auth/views.py

```diff
@@ -191,6 +192,7 @@ def get_oauth_profile(name, token=None, client=None):
     # Check if OIDC and can get useerInfo
     if _client.load_server_metadata().get('userinfo_endpoint'):
         resp = _client.userinfo()
+        groups = get_oidc_user_groups(resp['email'])
```

Comment:
`userinfo_endpoint` can exist for many different OICD clients. Should be explicit for `cern` client that can `get_oidc_user_groups`